packages/@uppy/xhr-upload/src/index.js

```diff
@@ -171,7 +171,11 @@ export default class XHRUpload extends BasePlugin {
       : Object.keys(meta) // Send along all fields by default.
 
     allowedMetaFields.forEach((item) => {
-      formData.append(item, meta[item])
+      if (Array.isArray(meta[item])) {
+        meta[item].forEach(subItem => formData.append(item, subItem))
```

Comment:
I think we need the brackets, otherwise it's going to overwrite the previous entry.
```suggestion
        meta[item].forEach(subItem => formData.append(`${item}[]`, subItem))
```

Reply:
it works ok, because item is something like `item_name[]` already and you can append two items with the same name
```js
const form = new FormData();
form.append('item[]', 1);
form.append('item[]', 2);
console.log([...form.entries()])
>  [Array(2), Array(2)]
0: (2) ['item[]', '1']
1: (2) ['item[]', '2']
```

with your suggestion, I think it will look like `item[][]` with extra brackets, because `item` is already a kind of array value

Reply:
Oh alright, could add a comment then please?

Reply:
@aduh95 done!